include/mgp.hpp

```diff
@@ -1572,6 +1585,14 @@ inline Id::Id(int64_t id) : id_(id) {}
 
 inline Graph::Graph(mgp_graph *graph) : graph_(graph) {}
 
+inline bool Graph::MustAbort() const { return must_abort(graph_); }
+
+inline void Graph::CheckMustAbort() const {
+  if (MustAbort()) {
+    throw MustAbortException("Query was asked to abort.");
+  }
```

Comment:
Python API has `CheckMustAbort` and `MustAbort`, where later doesn't return error, but 0 or 1. I think we can add that too. It does make sense to have one which doesn't throw.

Reply:
You can check in `py_module.cpp` 

```
int mgp_must_abort(mgp_graph *graph) {
  MG_ASSERT(graph->ctx);

  static_assert(noexcept(memgraph::query::MustAbort(*graph->ctx)));
  return memgraph::query::MustAbort(*graph->ctx) ? 1 : 0;
}
```

Reply:
There is already one, few lines above:

```
inline bool Graph::MustAbort() const { return must_abort(graph_); }
```

Reply:
line 1588